packages/eas-cli/src/utils/expoCli.ts

```diff
@@ -63,7 +63,8 @@ export async function expoCommandAsync(
   args: string[],
   { silent = false }: { silent?: boolean } = {}
 ): Promise<void> {
-  const expoCliPath = resolveFrom(projectDir, 'expo/bin/cli.js');
+  const expoCliPath =
+    silentResolveFrom(projectDir, 'expo/bin/cli') || resolveFrom(projectDir, 'expo/bin/cli.js');
```

Comment:
Prefer nullish coleasing:
```suggestion
    silentResolveFrom(projectDir, 'expo/bin/cli') ?? resolveFrom(projectDir, 'expo/bin/cli.js');
```